lib/index.js

```diff
@@ -8,7 +8,7 @@ const buildDebugMacroPlugin = require('./build-debug-macro-plugin');
 const buildStripClassCallcheckPlugin = require('./build-strip-class-callcheck-plugin');
 const injectBabelHelpers = require('./transforms/inject-babel-helpers').injectBabelHelpers;
 const debugTree = require('broccoli-debug').buildDebugCallback('ember-source:addon');
-const vmBabelPlugins = require('@glimmer/vm-babel-plugins');
+const { default: vmBabelPlugins } = require('@glimmer/vm-babel-plugins');
```

Comment:
```suggestion
const vmBabelPlugins = require('@glimmer/vm-babel-plugins').default;
```

Curious how come this was the only one that needs changing?

Reply:
everything else uses actual ESM.
The whole build sytsem got swapped out in https://github.com/glimmerjs/glimmer-vm/pull/1427 or https://github.com/glimmerjs/glimmer-vm/pull/1426

vm-babel-plugins is compiled to have:
```js
'use strict';

Object.defineProperty(exports, '__esModule', { value: true });

var node_module = require('node:module');

const require$1 = node_module.createRequire((typeof document === 'undefined' ? require('u' + 'rl').pathToFileURL(__filename).href : (document.currentScript && document.currentScript.src || new URL('index.cjs', document.baseURI).href)));
function defaultTo(value, defaultVal) {
  return value === undefined ? defaultVal : value;
}


// ...
 
exports.default = generateVmPlugins;
//# sourceMappingURL=index.cjs.map

```

https://www.npmjs.com/package/@glimmer/vm-babel-plugins?activeTab=code

Reply:
Resolving, as this is a correct change